test/edu-benefits/5490/config/benefitRelinquishment.unit.spec.jsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import ReactTestUtils from 'react-addons-test-utils';
+
+import { DefinitionTester, submitForm } from '../../../util/schemaform-utils.jsx';
+import formConfig from '../../../../src/js/edu-benefits/5490/config/form';
+
+describe('Edu 5490 benefitRelinquishment', () => {
+  const { schema, uiSchema } = formConfig.chapters.benefitSelection.pages.benefitRelinquishment;
+  it('should render', () => {
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          data={{}}
+          state={{
+            benefitSelection: {
+              uiSchema: {
+                benefit: {
+                  'ui:options': {
+                  }
+                }
+              }
+            }
+          }}
+          formData={{ relationship: 'child' }}
```

Comment:
I notice this isn't present in the other tests. Is it not needed? Perhaps more importantly, should it be needed?